### PR TITLE
i18nification

### PR DESCRIPTION
**Pull request by simkim**
implement #546

### CODE REVIEW

**Comment by simkim** [comment]
40% of work done

**Comment by simkim** [comment]
@andrew can you review the changes. I translated the majority of the code except usage which is not directly linked from front page

File: app/models/owner.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Owner < ApplicationRecord
   scope :created_after, ->(date) { where('created_at > ?', date) }
   scope :updated_after, ->(date) { where('updated_at > ?', date) }
 
-  scope :kind, ->(kind) { where(kind: kind) }
+  enum kind: { user: 'user', organization: 'organization' }
```
Comment (andrew) [inline app/models/owner.rb:10]: We still need the scope of kind here for the owners controller

**Review by andrew — CHANGES_REQUESTED** [review]
Just one small change otherwise LGTM

File: app/models/owner.rb (hunk 1 above)
Comment (simkim) [reply app/models/owner.rb:11]: I reverted the removal of the kind scope

**Review by andrew — APPROVED** [review]

**Pull request merged**